tests/util/test_yaml.py

```diff
@@ -372,6 +386,17 @@ def test_secrets_keyring(self):
         _yaml = load_yaml(self._yaml_path, yaml_str)
         self.assertEqual({'http': {'api_password': 'yeah'}}, _yaml)
 
+    def test_secrets_credstash(self):
```

Comment:
It doesn't look like your test uses any of the setup in this class. Please move it to be a standalone function in this file.